apps/jobs/utils.py

```diff
@@ -1,5 +1,122 @@
+            'max_submission_exceeded': True
+        }
+        return response_data, status.HTTP_200_OK
+
+    # Check for monthy submission limit
+    elif submissions_done_this_month_count >= max_submissions_per_month_count:
+        date_time_now = timezone.now()
+        next_month_start_date_time = date_time_now + datetime.timedelta(days=+30)
+        next_month_start_date_time = next_month_start_date_time.replace(
+            day=1, hour=0, minute=0, second=0, microsecond=0)
+        remaining_time = next_month_start_date_time - date_time_now
+
+        if submissions_done_today_count >= max_submissions_per_day_count:
+            response_data = {
+                'message': 'Both daily and monthly submission limits are exhausted!',
+                'remaining_time': remaining_time
+            }
+        else:
+            response_data = {
+                'message': 'You have exhausted this month\'s submission limit!',
+                'remaining_time': remaining_time
+            }
+        return response_data, status.HTTP_200_OK
+
+    # Checks if #today's successful submission is greater than or equal to max submission per day
+    elif submissions_done_today_count >= max_submissions_per_day_count:
+        date_time_now = timezone.now()
+        date_time_tomorrow = date_time_now + datetime.timedelta(1)
+        # Get the midnight time of the day i.e. 12:00 AM of next day.
+        midnight = date_time_tomorrow.replace(hour=0, minute=0, second=0)
+        remaining_time = midnight - date_time_now
+
+        response_data = {
+            'message': 'You have exhausted today\'s submission limit!',
+            'remaining_time': remaining_time
+        }
+        return response_data, status.HTTP_200_OK
+
+    else:
+        # calculate the remaining submissions from total submissions.
+        remaining_submission_count = max_submissions_count - submissions_done_count
+        # Calculate the remaining submissions for current month.
+        remaining_submissions_this_month_count = (max_submissions_per_month_count -
+                                                  submissions_done_this_month_count)
+        # Calculate the remaining submissions for today.
+        remaining_submissions_today_count = (max_submissions_per_day_count -
+                                             submissions_done_today_count)
+
+        remaining_submissions_this_month_count = min(remaining_submission_count,
+                                                     remaining_submissions_this_month_count)
+        remaining_submissions_today_count = min(remaining_submissions_this_month_count,
+                                                remaining_submissions_today_count)
+
+        response_data = {
+            'remaining_submissions_this_month_count': remaining_submissions_this_month_count,
+            'remaining_submissions_today_count': remaining_submissions_today_count,
+            'remaining_submissions': remaining_submission_count
```

Comment:
lets be consistent in the key names and change this key name to `remaining_submissions_count`